app/src/main/java/ch/ubique/notifyme/app/MainActivity.java

```diff
@@ -44,6 +56,10 @@ protected void onCreate(Bundle savedInstanceState) {
 		setContentView(R.layout.activity_main);
 
 		storage = Storage.getInstance(this);
+		if (storage.getDidAutoCheckout()) {
+			showMainFragment();
```

Comment:
This might cause an issue with savedInstanceState (i.e. DKA)

Reply:
Why?

Reply:
This adds the main fragment, but in case of a saved instance state the main fragment gets immediately replaced by any saved fragment.

Reply:
Wouldn't the old Fragment be restored in the super.onCreate(..) call? And then we add our Main Fragment after that?